node-api/handlers/beacon/backend.go

```diff
@@ -49,10 +48,6 @@ type GenesisBackend interface {
 	GenesisTime() (math.U64, error)
 }
 
-type RandaoBackend interface {
-	RandaoAtEpoch(slot math.Slot, epoch math.Epoch) (common.Bytes32, error)
-}
-
```

Comment:
I am not really we need there shotgun backend methods.
My idea is that:

- Logic should be in handlers, not backend
- Backend should contain the minimum common logic shared across handlers